### PR TITLE
Add New Shows and Season Premiers to Trakt Lists

### PR DESCRIPTION
**Pull request by Thraxis**
- Added New Shows starting in the next 30 days
- Added Season Premiers starting in the next 30 days
- Increased limit of returns for the other lists to 100 items.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: gui/slick/views/addShows_trendingShows.mako [hunk 1]
```diff
@@ -44,6 +44,8 @@
     <span style="margin-left:12px">Select Trakt List:</span>
     <select id="traktlistselection" class="form-control form-control-inline input-sm">
         <option value="anticipated" ${' selected="selected"' if traktList == "anticipated" else ''}>Most Anticipated</option>
+        <option value="newshow" ${' selected="selected"' if traktList == "newshow" else ''}>New Shows</option>
```
Comment (miigotu) [inline gui/slick/views/addShows_trendingShows.mako:47]: Series Premieres ?

**Comment by Thraxis** [comment]
I was using the same name that Trakt uses.  I can change it if you prefer Series Premiers

**Comment by miigotu** [comment]
just looks like there will be confusion between newShow and newshows

**Pull request merged**